server/routes/reporting/standard-report/definitions.js

```diff
@@ -376,6 +376,17 @@
         unsortable: true,
         exportLabel: 'Export raw data',
       },
+      'voir-dire': {
+        title: 'Panel result report (by trial)',
+        apiKey: 'PanelResultReport',
+        search: 'dateRange',
+        headings: [
+          'dateFrom',
+          'reportDate',
+          'dateTo',
+          'reportTime',
+        ],
+      },
```

Comment:
the trial type is showing as `Cri` for Criminal and `Civ` for Civil, this needs to be handled and mapped to the correct trial type.
the trial number also needs to be a clickable link

Reply:
Added